tenant/pkg/controller/tenantnamespace/tenantnamespace_controller.go

```diff
@@ -167,45 +170,96 @@ func (r *ReconcileTenantNamespace) Reconcile(request reconcile.Request) (reconci
 		UID:        instance.UID,
 	}
 
+	found := false
 	for _, each := range nsList.Items {
 		if each.Name == tenantNsName {
+			found = true
 			// Check OwnerReference
-			found := false
+			isOwner := false
 			for _, ownerRef := range each.OwnerReferences {
 				if ownerRef == expectedOwnerRef {
-					found = true
+					isOwner = true
 					break
 				} else if ownerRef.APIVersion == expectedOwnerRef.APIVersion && ownerRef.Kind == expectedOwnerRef.Kind {
 					// The namespace is owned by another TenantNamespace CR, fail the reconcile
 					err = fmt.Errorf("Namespace %v is owned by another %v TenantNamespace CR", each.Name, ownerRef)
 					return reconcile.Result{}, err
 				}
 			}
-			if !found {
+			if !isOwner {
 				log.Info("Namespace has been created without TenantNamespace owner", "namespace", each.Name)
 				// Obtain namespace ownership by setting ownerReference, and add annotation
 				err = r.updateNamespace(&each, &instance.Namespace, &expectedOwnerRef)
```

Comment:
This err is now ignored with this change

Reply:
fixed